tests/utils/test_dataset_combine.py

```diff
@@ -72,8 +72,8 @@ def _generate_dataset_with_collector_env(
         # Force None max_episode_steps
         env_spec = gym.make("CartPole-v1").spec
         assert env_spec is not None
-        env_spec.max_episode_steps = None
-        env = env_spec.make()
+        env_spec.max_episode_steps = None  # pyright: ignore[reportOptionalMemberAccess]
+        env = env_spec.make()  # pyright: ignore[reportOptionalMemberAccess]
```

Comment:
env_space is not None for sure, I think you can remove the pyright ignore

Reply:
I've removed all pyright inline ignores. Please do let me know if I should set the pyright ignore in the .toml file or if there is some other option to resolve this.